pkg/pillar/types/physicalioadapters.go

```diff
@@ -24,8 +23,19 @@ type PhysicalAddress struct {
 }
 
 // PhyIOUsagePolicy - Usage policy for the Adapter
+// This is constructed from
```

Comment:
Looks like comment got cut off.

Reply:
Done..fixed it.

 // PhyIOUsagePolicy - Usage policy for the Adapter
-// This is constructed from
+// This is constructed from api/proto/config/devmodel.proto PhyIOUsagePolicy
+// Keep the two structures consistent